telemetry/engagementProfileEvents.js

```diff
@@ -0,0 +1,116 @@
+export const profileEvents = {
+  // follow button
+  'profile.follow.unblock': {
+    engagement_type: 'general',
+  },
+  'profile.follow.unmute': {
+    engagement_type: 'general',
+  },
+  'profile.follow.unfollow': {
+    engagement_type: 'general',
+  },
+  'profile.follow.follow': {
+    engagement_type: 'general',
+  },
+  'profile.follow.withdraw-follow-request': {
+    engagement_type: 'general',
+  },
+  'profile.follow.follow-request': {
+    engagement_type: 'general',
+  },
+  // more button and more options dropdown
+  'profile.more.open': {
+    engagement_type: 'general',
+  },
+  'profile.more.open_in_original_site': {
+    engagement_type: 'general',
+  },
+  'profile.more.share_account': {
```

Comment:
How do you get to this action? It doesn't show up in the dropdown for me?

Reply:
I hardcoded this one to make it show in the dropdown in order to test